torch/_inductor/codegen/triton.py

```diff
@@ -2067,6 +2073,8 @@ def store(
         elif mode is None:
             line = f"tl.store({var} + ({indexing.index_str}), {value}, {indexing.mask_str})"
         elif mode == "atomic_add":
+            if isinstance(index, sympy.core.symbol.Symbol) and "(idx_m)" == index.name:
```

Comment:
Need to figure out the proper way to broadcast, dont have a good way over injecting the copy_shape into indexing generically